spec/controllers/api/v1/time_series_values_controller_spec.rb

```diff
@@ -14,6 +14,18 @@
         scenario: scenario
       )
     }
+    let!(:model_with_time_series) {
+      model = create(:model)
+      my_scenario = create(:scenario, model_id: model.id)
+      create_list(
+        :time_series_value,
+        5,
```

Comment:
I'm not sure if this number can be reduced, but it'd be nice to make it as low as possible. DB reads/writes usually have a big impact on how long the test suite takes to execute. This `let!` statement creates at least 7 records in the DB, and it's executed before *every* single test inside of this context.

Reply:
Yeah you're right. I'm going to reduce these numbers a bit.